src/test/videopress/edit.js

```diff
@@ -35,14 +36,21 @@ import {
 	selectAndOpenBlockSettings,
 	pressSettingInPanel,
 	pressSettingInPicker,
+	generateFetchMocks,
 } from './local-helpers/utils';
 
 setupCoreBlocks();
 
 beforeAll( () => {
 	// Register Jetpack blocks
-	setupJetpackEditor( { blogId: 1, isJetpackActive: true } );
+	setupJetpackEditor( {
+		blogId: 1,
+		isJetpackActive: true,
+	} );
 	registerJetpackBlocks( DEFAULT_PROPS );
+
+	// Mock request reponses
+	setupApiFetch( generateFetchMocks() );
```

Comment:
Why this now required due to the RN upgrade?

Reply:
Yep, it looks related to that change. Without this change, the tests fail with the following error:
```
TypeError: Cannot read property 'then' of undefined

      53 | 			method: 'POST',
      54 | 			body: fetchParams.body,
    > 55 | 		} ).then( response => {
         | 		   ^
      56 | 			const { metadata_token } = response;
      57 | 			if ( ! metadata_token ) {
      58 | 				console.warn( 'Token is not achievable' ); // eslint-disable-line no-console
```

I understand that when `Promise` was polyfilled in Jest tests, we didn't need to explicitly mock API responses (i.e. `api-fetch` library).